configs/kvstore_and_fw_candidate_on_sd.json

```diff
@@ -62,9 +62,7 @@
                     "",
                     "NRF52840_DK: Please note the config is different. The kvstore region is at the end of flash.",
                     "Setting `target.static_memory_defines` as `true` is an alternative for the workaround",
-                    "https://github.com/ARMmbed/mbed-os/pull/11510 . The PR in question is waiting for approval",
-                    "",
-                    "Reason for removing labels and some components like PSA or LPTICKER is to fit bootloader into 32kB"
+                    "https://github.com/ARMmbed/mbed-os/pull/11510 . The PR in question is waiting for approval"
```

Comment:
This mbed-os pull was closed already, please update this link/description.

Reply:
Updated to point on the issue which is still open.